src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -27,11 +28,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
-        long driverId = Long.parseLong(req.getParameter("driver_id"));
+        long driverId = Long.parseLong(req.getParameter(DRIVER_ID));
```

Comment:
```suggestion
        long driverId = Long.parseLong(req.getParameter("driver_id"));
```

I think it's better to leave it as it was, because parameter from jsp form and session parameter are different parameters.